source/identical.js

```diff
@@ -27,5 +27,23 @@ import _curry2 from './internal/_curry2.js';
  *      R.identical(0, -0); //=> false
  *      R.identical(NaN, NaN); //=> true
  */
-var identical = _curry2(_objectIs);
+var identical = function(a, b) {
+  switch (arguments.length) {
+    case 0:
+      return identical; 
+    case 1:
+      return (function(){
+        return function unaryIdentical(_b) {
+          switch (arguments.length) {
+            case 0:
+              return unaryIdentical;
+            default:
+              return _objectIs(a, _b);
+          }
+        };
+      }());
+    case 2:
```

Comment:
Would it make sense for `case 2` to be the default case instead? It would handle the case were additional arguments are given (even though they would be ignored).

Reply:
This makes sense. It's fixed now in [semmel:patch-1](https://github.com/semmel/semmel-ramda/tree/patch-1)